api/test/OpenTelemetry/Logging/CoreSpec.hs

```diff
@@ -43,7 +52,9 @@ spec = describe "Core" $ do
             }
       setGlobalLoggerProvider lp
       glp <- getGlobalLoggerProvider
-      glp `shouldBe` lp
+      fmap TestLogProcessor (loggerProviderProcessors glp) `shouldSatisfy` null
+      loggerProviderResource glp `shouldBe` loggerProviderResource lp
+      loggerProviderAttributeLimits glp `shouldBe` loggerProviderAttributeLimits lp
```

Comment:
Testing each field separately is similar to testing the `LoggerProvider` as a whole, but it makes more sense now that `LogProcessor`s are part of `LoggerProvider`.